docs/bigquery/snippets.py

```diff
@@ -440,64 +457,167 @@ def test_table_exists(client, to_delete):
     assert not table_exists(client, dataset.table('i_dont_exist'))
 
 
-def test_update_table_simple(client, to_delete):
-    """Patch a table's metadata."""
-    DATASET_ID = 'update_table_simple_dataset_{}'.format(_millis())
-    TABLE_ID = 'update_table_simple_table_{}'.format(_millis())
-    dataset = bigquery.Dataset(client.dataset(DATASET_ID))
-    dataset.description = ORIGINAL_DESCRIPTION
+def test_update_table_description(client, to_delete):
+    """Update a table's description."""
+    dataset_id = 'update_table_description_dataset_{}'.format(_millis())
+    table_id = 'update_table_description_table_{}'.format(_millis())
+    dataset = bigquery.Dataset(client.dataset(dataset_id))
     client.create_dataset(dataset)
     to_delete.append(dataset)
 
-    table = bigquery.Table(dataset.table(TABLE_ID), schema=SCHEMA)
-    table.description = ORIGINAL_DESCRIPTION
+    table = bigquery.Table(dataset.table(table_id), schema=SCHEMA)
+    table.description = "Original description."
```

Comment:
Likewise, use an `original_description` variable to avoid repeating the literal.

Reply:
I'm okay with this being a literal to make it 100% clear the expected data type in the snippet itself, but it should use single-quotes.